services/docker-compose.yml

```diff
@@ -236,6 +238,41 @@ services:
       - default
       - interactive_services_subnet
 
+  webserver-garbage-collector:
+    image: ${DOCKER_REGISTRY:-itisfoundation}/webserver:${DOCKER_IMAGE_TAG:-latest}
+    init: true
+    hostname: "{{.Node.Hostname}}-{{.Service.Name}}-{{.Task.Slot}}"
+    environment:
+      <<: *webserver-environment
+      WEBSERVER_GARBAGE_COLLECTOR: '{"GARBAGE_COLLECTOR_INTERVAL_S": 30}'
+    env_file:
+      - ../.env
+      - ../.env-webserver-garbage-collector
```

Comment:
so this one overrides what is in .env? 
So I guess we should review these contents soon.

Reply:
yes, that what I suggest in the note above as well ... but I want to do it carefully and taking into account the devops repo

Reply:
We definately need to clean up the env-var handling. They come in from all over the place, and it is hard to detangle since some env-vars (IPs etc.) are overwritten by makefiles, overwritten from the simcore repo, etc.

Reply:
I feel it would be clearer to just put all the variable without using the *webserver-environment (so, having two times the same thing) and set theme here to null, not in the env-webserver-garbage-collector as it adds another layer in top of 3 another layers.. It would be repeating but easier to understand